<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectFour_CSCI440Project_v01Class</class>
 <widget class="QMainWindow" name="ConnectFour_CSCI440Project_v01Class">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>805</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>ConnectFour_CSCI440Project_v01</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="boardBG">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>801</width>
      <height>791</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ConnectFour_CSCI440Project_v01.qrc">:/ConnectFour_CSCI440Project_v01/Resources/board_BG.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="restartBtn">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>700</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="redBtn">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ConnectFour_CSCI440Project_v01/Resources/red.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="yellowBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ConnectFour_CSCI440Project_v01/Resources/yellow.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sixOne">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>570</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="oneOne">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>570</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fourZero">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>680</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sixTwo">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>460</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fourFive">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="twoZero">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>680</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="oneZero">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>680</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="threeOne">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>570</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fiveFive">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="oneTwo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>460</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fourOne">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>570</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="threeFive">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="threeFour">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="zeroOne">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fourThree">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>340</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sixFive">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>120</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sixZero">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>680</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sixThree">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>340</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sixFour">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>230</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="oneFive">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="zeroThree">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fourFour">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>230</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fiveFour">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>230</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fiveOne">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>570</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="zeroFive">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="twoOne">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>570</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="zeroTwo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="twoThree">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="threeZero">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>680</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="oneThree">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="twoTwo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>460</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fiveZero">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>680</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="threeThree">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fourTwo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>460</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="zeroFour">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="twoFive">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fiveThree">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>340</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="threeTwo">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>460</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="oneFour">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="zeroZero">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fiveTwo">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>460</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="twoFour">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="col1">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>0</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 86, 193);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="col2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 86, 193);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="col3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 86, 193);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="col4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 86, 193);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="col5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 86, 193);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="col6">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 86, 193);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="col7">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 86, 193);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 0);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>sixOne</zorder>
   <zorder>sixZero</zorder>
   <zorder>sixThree</zorder>
   <zorder>sixFour</zorder>
   <zorder>sixTwo</zorder>
   <zorder>sixFive</zorder>
   <zorder>restartBtn</zorder>
   <zorder>oneOne</zorder>
   <zorder>fourZero</zorder>
   <zorder>fourFive</zorder>
   <zorder>twoZero</zorder>
   <zorder>oneZero</zorder>
   <zorder>threeOne</zorder>
   <zorder>fiveFive</zorder>
   <zorder>oneTwo</zorder>
   <zorder>fourOne</zorder>
   <zorder>threeFive</zorder>
   <zorder>threeFour</zorder>
   <zorder>zeroOne</zorder>
   <zorder>fourThree</zorder>
   <zorder>oneFive</zorder>
   <zorder>zeroThree</zorder>
   <zorder>fourFour</zorder>
   <zorder>fiveFour</zorder>
   <zorder>fiveOne</zorder>
   <zorder>zeroFive</zorder>
   <zorder>twoOne</zorder>
   <zorder>zeroTwo</zorder>
   <zorder>twoThree</zorder>
   <zorder>threeZero</zorder>
   <zorder>oneThree</zorder>
   <zorder>twoTwo</zorder>
   <zorder>fiveZero</zorder>
   <zorder>threeThree</zorder>
   <zorder>fourTwo</zorder>
   <zorder>zeroFour</zorder>
   <zorder>twoFive</zorder>
   <zorder>fiveThree</zorder>
   <zorder>threeTwo</zorder>
   <zorder>oneFour</zorder>
   <zorder>zeroZero</zorder>
   <zorder>fiveTwo</zorder>
   <zorder>twoFour</zorder>
   <zorder>redBtn</zorder>
   <zorder>yellowBtn</zorder>
   <zorder>boardBG</zorder>
   <zorder>col1</zorder>
   <zorder>col2</zorder>
   <zorder>col3</zorder>
   <zorder>col4</zorder>
   <zorder>col5</zorder>
   <zorder>col6</zorder>
   <zorder>col7</zorder>
   <zorder>scoreLabel</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ConnectFour_CSCI440Project_v01.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>yellowBtn</sender>
   <signal>clicked()</signal>
   <receiver>ConnectFour_CSCI440Project_v01Class</receiver>
   <slot>on_yellowBtn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>868</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redBtn</sender>
   <signal>clicked()</signal>
   <receiver>ConnectFour_CSCI440Project_v01Class</receiver>
   <slot>on_redBtn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>836</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restartBtn</sender>
   <signal>clicked()</signal>
   <receiver>ConnectFour_CSCI440Project_v01Class</receiver>
   <slot>on_restartBtn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>920</x>
     <y>717</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>col1</sender>
   <signal>clicked()</signal>
   <receiver>ConnectFour_CSCI440Project_v01Class</receiver>
   <slot>on_arrowBtn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>-13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>col2</sender>
   <signal>clicked()</signal>
   <receiver>ConnectFour_CSCI440Project_v01Class</receiver>
   <slot>on_arrowBtn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>-22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>col3</sender>
   <signal>clicked()</signal>
   <receiver>ConnectFour_CSCI440Project_v01Class</receiver>
   <slot>on_arrowBtn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>714</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>col4</sender>
   <signal>clicked()</signal>
   <receiver>ConnectFour_CSCI440Project_v01Class</receiver>
   <slot>on_arrowBtn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>col5</sender>
   <signal>clicked()</signal>
   <receiver>ConnectFour_CSCI440Project_v01Class</receiver>
   <slot>on_arrowBtn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>840</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>col6</sender>
   <signal>clicked()</signal>
   <receiver>ConnectFour_CSCI440Project_v01Class</receiver>
   <slot>on_arrowBtn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>624</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>863</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>col7</sender>
   <signal>clicked()</signal>
   <receiver>ConnectFour_CSCI440Project_v01Class</receiver>
   <slot>on_arrowBtn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>730</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>867</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_restartBtn_clicked()</slot>
  <slot>on_arrowBtn_clicked()</slot>
 </slots>
</ui>
